compiler/semantic/parallelize.go

```diff
@@ -227,11 +227,12 @@ func parallelize(p ast.Proc, N int, inputSortField field.Static, inputSortRevers
 
 	seq := ensureSequentialProc(p)
 	orderSensitiveTail := true
+loop:
 	for i := range seq.Procs {
 		switch seq.Procs[i].(type) {
 		case *ast.SortProc, *ast.GroupByProc:
 			orderSensitiveTail = false
-			break
+			break loop
```

Comment:
Unrelated to this bug, but thought it should be fixed.

Reply:
It should! But it's unrelated to the other change in this PR, so a separate PR would be better.